modules/twinkleblock.js

```diff
@@ -774,6 +774,14 @@ Twinkle.block.blockPresetsInfo = {
 		reason: '{{uw-botublock}} <!-- Username implies a bot, soft block -->',
 		summary: 'You have been indefinitely blocked from editing because your [[WP:U|username]] indicates this is a [[WP:BOT|bot]] account, which is currently not approved'
 	},
+	'uw-botuhblock': {
+		autoblock: true,
+		expiry: 'infinity',
+		forRegisteredOnly: true,
+		nocreate: true,
+		reason: '{{uw-botuhblock}} <!-- Username implies a bot, [[WP:NOTHERE]], hard block -->',
+		summary: 'You have been indefinitely blocked from editing because your [[WP:U|username]] contains the suffix "-bot" and your contributions do not suggest that you are here to contribute constructively to the encyclopedia.'
```

Comment:
Why not just 
```suggestion
		summary: 'You have been indefinitely blocked from editing because your username is a blatant violation of the [[WP:U|username policy]].'
````

It's an edit summary, doesn't need to be everything.  Or just take the same text from the softer version
```suggestion	
		summary: 'You have been indefinitely blocked from editing because your [[WP:U|username]] indicates this is a [[WP:BOT|bot]] account, which is currently not approved.'
```